packages/terra-form-select/src/_Frame.module.scss

```diff
@@ -120,14 +120,17 @@
     padding: var(--terra-select-toggle-padding, 0 0.42857rem);
   }
 
-  .tags {
+  .content {
```

Comment:
I'm struggling to find a good name for this content region, if anyone has any ideas let me know. This content region is the wrapper around the input and selected values for tag, multiple, and combobox variants. The default variant does not have this wrapper.

Reply:
`.input-container` is what comes to mind. I'm cool with `.content`.

Reply:
if you want to be more specific, maybe `non-default-content`?